backend/src/main/java/de/ffmjava/capstone/backend/stock/StockService.java

```diff
@@ -16,6 +18,15 @@ public List<StockItem> getAllStockItems() {
         return repository.findAll();
     }
 
+    public boolean deleteStockItem(String id) throws ResponseStatusException {
```

Comment:
most certain that i dont see the reason, but can u explain why boolean?

Reply:
Basically to make it testable in unittests